keyboards/coban/pad9a/info.json

```diff
@@ -0,0 +1,61 @@
+{
+    "keyboard_name": "Coban Pad 9A",
+    "name": "Coban Pad 9A",
+    "url": "https://cobanstationery.com",
+    "maintainer": "Coban Stationery",
+    "manufacturer": "Coban Stationery",
+    "usb": {
+        "pid": "0xCC9A",
+        "vid": "0xCB3A",
+        "device_version": "1.2.0"
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP22", "GP2", "GP19", "GP16", "GP10", "GP18", "GP17", "GP11"]
+        ]
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "rgb_matrix": true
```

Comment:
```suggestion
        "rgb_matrix": true,
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true
```

Reply:
I added `bootmagic` and updated feature order as your suggestion.